src/tests/system/tests/test_sss_cache.py

```diff
@@ -43,3 +46,69 @@ def test_sss_cache__cache_expire_message(client: Client):
     for cmd in ("sss_cache -U", "sss_cache -G", "sss_cache -E", "sss_cache --user=nonexisting"):
         res = client.host.ssh.run(cmd)
         assert "No domains configured, fatal error!" not in res.stdout
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.cache
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_sss_cache__background_refresh(client: Client, provider: GenericProvider):
+    """
+    :title: Ensuring ldb cache data is refreshed correctly
+    :setup:
+        1. Create provider user
+        2. Create provider group
+        3. Create provider netgroup
+        4. Perform 'getent' on user, group and netgroup
+    :steps:
+        1. Set "entry_cache_timeout" to 30 and "refresh_expired_interval" to 60
+        2. Restart SSSD
+        3. Search for user, group and netgroup lastUpdate and dataExpireTimestamp in the ldb database
+        4. Wait 60 seconds and search for all timestamp in the cache again
+    :expectedresults:
+        1. SSSD is configured
+        2. SSSD restarts successfully
+        3. dataExpireTimestamp value equals lastUpdate + entry_cache_timeout
+        4. User, group and netgroup 'lastUpdate' time has been refreshed
+    """
+    user = provider.user("test_user").add()
+    provider.group("test_group").add().add_member(user)
+    provider.netgroup("test_netgroup").add().add_member(user=user)
+
+    domain = client.sssd.default_domain
+    entry_cache_timeout = 4
+    refresh_expired_interval = 8
+
+    client.sssd.domain["entry_cache_timeout"] = str(entry_cache_timeout)
+    client.sssd.domain["refresh_expired_interval"] = str(refresh_expired_interval)
+
+    client.sssd.restart()
+    client.tools.getent.passwd(f"test_user@{domain}")
+    client.tools.getent.group(f"test_group@{domain}")
+    client.tools.getent.netgroup(f"test_netgroup@{domain}")
```

Comment:
These three lines are explained in the setup phase, but they come after step 2, which I don't think should happen. Please move them several lines above to follow the order defined in the comments.